src/encoded/static/components/item-pages/PublicationView.js

```diff
@@ -7,11 +7,14 @@ import _ from 'underscore';
 import { console, object } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
 import { formatPublicationDate } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/LocalizedTime';
 
+import { PageTitleContainer, TitleAndSubtitleUnder, OnlyTitle, StaticPageBreadcrumbs, pageTitleViews } from './../PageTitle';
+import { Wrapper as ItemHeaderWrapper, TopRow, MiddleRow, BottomRow } from './components/ItemHeader';
```

Comment:
Codacy found an issue: ['ItemHeaderWrapper' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5411404)